src/data/users/angamanga.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Anna Iosif",
+  "github": "angamanga",
```

Comment:
Same thing here, this line should be: `"github": "Angamanga",`

...and this file itself must be renamed to `Angamanga.json`!